provider/cmd/pulumi-resource-cloudflare/schema.json

```diff
@@ -3585,14 +3645,14 @@
                     "items": {
                         "type": "string"
                     },
-                    "description": "Check for presence of specified cookies, without including their actual values.\n"
+                    "description": "Check for presence of specified HTTP headers, without including their actual values.\n"
```

Comment:
I think this is a regression

Reply:
Yep, and the fix at 3675 is a correction. My guess is that we were (and still are) mis-parsing the nested field, so the doc piece that appears later is overwriting the docs piece that appears earlier (as opposed to vice versa).